signingscript/docker.d/passwords.yml

```
@@ -200,6 +200,11 @@ in:
              {"$eval": "AUTOGRAPH_GPG_PASSWORD"},
              ["autograph_gpg"]
             ]
+          - ["https://autograph-external.prod.autograph.services.mozaws.net",
```

I might be wrong. My understand was that we would talk to the staging instance of autograph. If that's actually the case, we should replace this URL by: 

```suggestion
          - ["https://autograph-external.stage.autograph.services.mozaws.net",
```

---

Quite right!